sdk/src/main/scala/com/horizen/account/mempool/AccountMemoryPool.scala

```diff
@@ -67,11 +67,11 @@ class AccountMemoryPool(unconfirmed: TrieMap[String, SidechainTypes#SCAT])
 
   def isTransactionCompatible(tx: SidechainTypes#SCAT, txList: Seq[SidechainTypes#SCAT]) : Boolean = {
     val from = tx.getFrom
-    val nonce = new ByteArrayWrapper(tx.getNonce)
+    val nonce = new ByteArrayWrapper(tx.getNonce.toByteArray)
```

Comment:
here we can use BigInteger type directly.
so
`val nonce: BigInteger = tx.getNonce`

and below:
`x.getFrom == from && x.getNonce == nonce`

previously we used a wrapper, because byte array is not comparable by default.